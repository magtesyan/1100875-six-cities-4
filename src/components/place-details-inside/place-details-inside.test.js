import renderer from "react-test-renderer";
import React from "react";

import PlaceDetailsInside from "./place-details-inside.jsx";

const features = [
  `Wi-Fi`,
  `Washing machine`,
  `Towels`,
  `Heating`,
  `Coffee machine`,
  `Baby seat`,
  `Kitchen`,
  `Dishwasher`,
];

it(`Features in PlaceDetailsInside are not rendered correctly`, () => {
  const tree = renderer
    .create(<PlaceDetailsInside
      features={features}
    />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
