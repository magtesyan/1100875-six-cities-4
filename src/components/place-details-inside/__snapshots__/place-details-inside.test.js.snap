// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features in PlaceDetailsInside are not rendered correctly 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Wi-Fi
    </li>
    <li
      className="property__inside-item"
    >
      Washing machine
    </li>
    <li
      className="property__inside-item"
    >
      Towels
    </li>
    <li
      className="property__inside-item"
    >
      Heating
    </li>
    <li
      className="property__inside-item"
    >
      Coffee machine
    </li>
    <li
      className="property__inside-item"
    >
      Baby seat
    </li>
    <li
      className="property__inside-item"
    >
      Kitchen
    </li>
    <li
      className="property__inside-item"
    >
      Dishwasher
    </li>
  </ul>
</div>
`;
